cl_dll/hud_strafeguide.cpp

```diff
@@ -120,24 +119,30 @@ void CHudStrafeGuide::Update(struct ref_params_s *pparams)
 
 	bool onground = pparams->onground;
 	double accelCoeff = onground ? pparams->movevars->accelerate : pparams->movevars->airaccelerate;
-	double frictionCoeff = pparams->movevars->friction;
-	double speedCap = onground ? pparams->movevars->maxspeed : min(pparams->movevars->maxspeed, 30);
+	//TODO: grab the entity friction from somewhere. pparams->movevars->friction is sv_friction
```

Comment:
when i am still not mistaken: playermove_s stores alteration from gravity / friction or conveyor belts